vars/OpenBSD.yml

```diff
@@ -1,6 +1,3 @@
-__editors_map:
+editor_package_names:
   vim: vim--no_x11
-  # emacs--no_x11 does not work here. because it matches emacs-24.5p2-no_x11
-  # and emacs-21.4p32-no_x11. #97 would fix the issue.
-  # https://github.com/ansible/ansible-modules-extras/issues/97
-  emacs: emacs-24.5p2-no_x11
+  emacs: emacs-24.5p5-no_x11
```

Comment:
the problem described in the comment has not been resolved. the point is, you should avoid hard-coded versions where possible.

would you remove issues in comments after you understand the issues? this is not yet officially documented, but we have agreed to the rule in a review meeting when you are absent.